README.md

```diff
@@ -24,9 +24,9 @@ composer phpstan
 Controller/ApplController.php is excluded 
 Please take some time to refactor the deprecated functions to use dependency injection instead of getting services from the container
 
-If you want to regenerate a baseline run this command:
+If you want to regenerate a PHPStan baseline run this command:
 ```
-vendor/bin/phpstan analyse --configuration phpstan.neon.dist src/ tests/ --generate-baseline
+vendor/bin/phpstan analyse --configuration phpstan.neon.dist src/ tests/ src-user/ --generate-baseline
```

Comment:
paths are defined in the phpstan.neon.dist so this should give you the same result: 

vendor/bin/phpstan analyse --generate-baseline